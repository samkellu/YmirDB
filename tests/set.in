set a 123
list entries
list keys
set b 22 a
list entries
list keys
bye
