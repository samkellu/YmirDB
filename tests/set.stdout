==858== Memcheck, a memory error detector
==858== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==858== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==858== Command: ./ymirdb
==858== 
==858== error calling PR_SET_PTRACER, vgdb might block
> not permitted

> no such key

> ok

> []

> ok

> [123 33]

> ok

> [a 22]

> [123 33]

> bye
==858== 
==858== HEAP SUMMARY:
==858==     in use at exit: 0 bytes in 0 blocks
==858==   total heap usage: 45 allocs, 45 frees, 2,000 bytes allocated
==858== 
==858== All heap blocks were freed -- no leaks are possible
==858== 
==858== For lists of detected and suppressed errors, rerun with: -s
==858== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
